packages/rum-core/src/domain/resource/resourceUtils.ts

```diff
@@ -108,6 +110,11 @@ export function computeResourceEntryDetails(entry: RumPerformanceResourceTiming)
     first_byte: formatTiming(startTime, requestStart, responseStart),
   }
 
+  // Make sure a worker processing time is recorded
+  if (workerStart < fetchStart) {
+    details.worker = formatTiming(startTime, workerStart, fetchStart)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Can you check in which case this situation can happen and maybe add a unit test to cover it.

Reply:
Yes, unit tests would be nice here.